core/lib/spree/testing_support/preferences.rb

```diff
@@ -14,7 +14,9 @@ module Preferences
       #
       # @deprecated
       def reset_spree_preferences(&config_block)
-        Spree::Config.instance_variables.each { |iv| Spree::Config.remove_instance_variable(iv) }
+        Spree::Config.instance_variables.
+          reject { |iv| iv == :@events_configuration }.
```

Comment:
I'm not sure about this. What if some specs need to reset that configuration? It's applicable? How do people know that all the preferences except that one will be reset when calling `reset_spree_preferences` in their code?

Reply:
This helper method is deprecated, so it won't be around for long. We don't use it anymore in our codebase, but we have some specs that verify its behavior. I'm not sure if creating a better workaround for making it work properly with the new event mappings is worth the effort at this point.

Reply:
Oh right, it's deprecated. 👍